data/categories.txt

```diff
@@ -275,7 +275,7 @@ fa:پارکینگ
 ca:Aparcament
 
 # Generic only! _shop_ category, will be present in all! _shop_ types.
-@shop
+@category_shop
```

Comment:
There is a bit of confusion here.
The `@` grouping could be used just to avoid duplicating search synonyms for similar feature types. These `@groups` don't necessarily show up as pre-defined search categories on the search screen in the app. And hence they don't have corresponding translations in `strings.txt`.

`@shop` is one such grouping, others are
```
@gambling
@waterbody
@charging_station
```

Maybe makes sense to distinguish these 2 cases by prepending `category_` to pre-defined UI-selectable categories only?

Reply:
... and add a comment that the "category_" prefix has a special meaning and processing

Reply:
@pastk good eye, thanks. Can you please help with the fix?

Reply:
Specifically, the current `@category_shop` seems redundant to me.
```
# Generic only! _shop_ category, will be present in all! _shop_ types.
@category_shop
en:Shop|Store|U+1F3EA|U+1F3EC
ru:4Магазин
```
Basically its applied to all `shop-*` types. What is the value of searching for / displaying **any/all** kinds of shops around you?

Note that its different from an UI pre-defined
```
@category_shopping
en:4Shopping
ru:3Шоппинг|4Покупки
```
which is applied more selectively to places where people tend to go shopping as a kind of activity (malls, department stores, jewelery, clothes...) - this one makes sense.

I suggest to remove `@category_shop` completely. WDYT?
@biodranik @vng

Reply:
There was a reason why it was added. Did you check git history?

Reply:
You are right: https://github.com/organicmaps/organicmaps/pull/5323

@vng so without this `@shop` addittion e.g. `wine shop` query will work worse?
I had an impression before that category matching uses just one synonym, not an arbitrary combination of them.

Now when I search for "wine shop" first results are `shop-wine`, but then they're followed by all sorts of shops >2000km away from me and my viewport - if they have a "shop" bit in their name.

And when I search for "магазин вина" first results are good too, but then e.g. a "Loft designe" furniture store follows and "Безопасность" outdoor shop? The worst thing is that they're in the same city so the map view is polluted by unrelated search results..

Reply:
- OK if you want to revert category_shop -> shop grouping tag, but don't remove it (at least now).
- Make an issue if you see some strange search results with categories. Matching sometimes is fancy and not obvious and I constantly improve it, but definitely generic _shop_ should exist.

Reply:
- https://github.com/organicmaps/organicmaps/issues/7644